docs/_content/standaard/standaard.md

```diff
@@ -454,6 +453,29 @@ gerelateerde objecten wordt begrepen:
 
 Een deelzaak KAN vernietigd worden zonder dat de hoofdzaak vernietigd wordt.
 
+#### Data filteren bij de bron
+
+Het autorisatiecomponent (AC) legt op het niveau van `zaaktype` vast welke
+operaties mogelijk en wat de maximale vertrouwelijkheidaanduiding is voor een
+consumer.
+
+Het ZRC MAG ENKEL zaken ontsluiten waarvan:
+
+* het zaaktype voorkomt in de autorisaties van de consumer
+* de vertrouwelijkheidaanduiding lager of gelijk aan de maximale
+  vertrouwelijkheidaanduiding is voor het betreffende zaaktype
+
+De API-specificatie legt vast welke scopes nodig zijn voor welke operaties.
+Een provider MOET operaties blokkeren op zaken waarvan de nodige scopes niet
+toegekend zijn voor het zaaktype van de betreffende zaak.
+
+Indien een operatie niet toegelaten is, dan MOET de provider met een
+`HTTP 403` foutbericht antwoorden.
+
+Een consumer kan de vlag `heeftAlleAutorisaties` gezet hebben - indien deze
```

Comment:
Deze begrijp ik niet. Wat is dat voor vlag? En waar staat die?

Reply:
Het concept `Applicatie`, waar de `autorisaties` aan hangen. Indien je een applicatie alles wil laten doen, dan hoef je niet `autorisaties` aan te maken voor alle zaaktypen en alles aanzetten, maar je kan het shortcutten door op de applicatie zelf `heeft_all_autorisaties` aan te zetten.

Reply:
Verduidelijking toegevoegd in de tekst